### ISSUE TITLE
Changes to the feedback tag list have an odd impact on URLs

### ISSUE THREAD

**Issue by jsorva** [apluslms/mooc-jutut#91]
I’ll just explain how to reproduce this:

1. Use the feedback-filtering form to search for messages with a particular feedback tag. Here’s an example URL from the O1 course: 
https://jutut.cs.aalto.fi/manage/55/feedbacks/?contains_text=on&tags_1_10=I232&order_by=-timestamp
Make a bookmark with this URL. (Side note: I use that example tag to mark urgent stuff that I should deal with, so I have the above link bookmarked.)

2. Revisit the URL via the bookmark, for example. Notice how the selected tag is highlighted in the filtering form’s UI, as it should be.

3. Go to Manage Feedback tags and add some random feedback tag. Let’s say we add a tag called "aaaa" at the top of the list.

4. Go back to the above URL. Notice how the selected tag is *not* highlighted any more. 

5. Click the tag in the form again and search. Notice that the URL is now different than before the "aaaa" tag was added. (The "tags" parameter has changed. Presumably the added feedback tag impacted on the numbering of the other, existing tags.)

The same in simpler terms: whenever I add a feedback tag (or another staff member does), my Jutut bookmarks (partially) break.

**Comment by Mankro** [comment]
I wonder why the tag query parameter has been implemented like that with the ordinal numbers. I think the tags should also have unique textual slugs.
